src/main/java/spoon/reflect/declaration/CtTypeInformation.java

```diff
@@ -180,15 +203,20 @@ public interface CtTypeInformation {
 	Collection<CtExecutableReference<?>> getAllExecutables();
 
 	/**
-	 * @return the type erasure, which is computed by the java compiler to ensure that no new classes are created for parametrized types so that generics incur no runtime overhead.
-	 * See https://docs.oracle.com/javase/tutorial/java/generics/erasure.html
+	 * This method returns a reference to the type with generic parameters removed.
+	 * <p>
+	 * For example the generic type {@code String} of {@code List<String>} is not known at
+	 * runtime, therefore it will be erased to {@code List}.
```

Comment:
That reasoning is not quite sound sadly, but maybe good enough.

Reply:
What about:
```java
/**
 * This method returns a reference to the erased type.
 * <p>
 * For example, this will return {@code List} for {@code List<String>},
 * or {@code Enum} for the type parameter {@code E} in the enum
 * declaration.
 *
 * @return a reference to the erased type
 * @see <a href="https://docs.oracle.com/javase/specs/jls/se20/html/jls-4.html#jls-4.6">Type Erasure</a>
 */
```